src/Matterhorn/Config.hs

```diff
@@ -47,8 +49,9 @@ userSyntaxPlaceholderName = "USER_SYNTAX"
 defaultSkylightingPaths :: IO [FilePath]
 defaultSkylightingPaths = do
     xdg <- xdgSyntaxDir
+    cabalDataFiles <- Paths.getDataFileName syntaxDirName
     adjacent <- getBundledSyntaxPath
-    return [xdg, adjacent]
+    return [xdg, cabalDataFiles, adjacent]
```

Comment:
Please put `cabalDataFiles` last in this list.

Reply:
Done.